python/paddle/v2/framework/layer_helper.py

```diff
@@ -0,0 +1,151 @@
+from paddle.v2.framework.graph import Variable, g_program
+import paddle.v2.framework.core as core
+import copy
+import itertools
+
+
+def unique_name(prefix):
+    uid = core.unique_integer()  # unique during whole process.
+    return "_".join([prefix, str(uid)])
+
+
+class LayerHelper(object):
+    def __init__(self, layer_type, **kwargs):
+        self.kwargs = kwargs
+        self.layer_type = layer_type
+        name = self.kwargs.get('name', None)
+        if name is None:
+            self.kwargs['name'] = unique_name(self.layer_type)
+
+    @property
+    def name(self):
+        return self.kwargs['name']
+
+    @property
+    def program(self):
+        return self.kwargs.get('program', g_program)
+
+    def append_op(self, *args, **kwargs):
+        return self.program.current_block().append_op(*args, **kwargs)
+
+    @property
+    def multiple_input(self):
```

Comment:
A layer may have more than one `input`, so this function shall take a `str` to indicate which `input` we want to get.